src/tests/class_tests/openms/source/MRMFeatureSelector_test.cpp

```diff
@@ -47,23 +54,30 @@ START_TEST(MRMFeatureSelector, "$Id$")
 /////////////////////////////////////////////////////////////
 /////////////////////////////////////////////////////////////
 
-MRMFeatureSelector* ptr = 0;
-MRMFeatureSelector* null_ptr = 0;
+MRMFeatureSelectorScore* ptr = 0;
+MRMFeatureSelectorScore* null_ptr = 0;
+
+const String features_path = OPENMS_GET_TEST_DATA_PATH("MRMFeatureSelector_150601_0_BloodProject01_PLT_QC_Broth-1_1.featureXML");
+
+FeatureMap feature_map;
+FeatureXMLFile feature_file;
+std::cout << feature_file.getOptions().getLoadSubordinates() << std::endl;
+feature_file.load(features_path, feature_map);
 
-START_SECTION(MRMFeatureSelector())
+START_SECTION(MRMFeatureSelectorScore())
 {
-  ptr = new MRMFeatureSelector();
+  ptr = new MRMFeatureSelectorScore();
   TEST_NOT_EQUAL(ptr, null_ptr)
 }
 END_SECTION
 
-START_SECTION(~MRMFeatureSelector())
+START_SECTION(~MRMFeatureSelectorScore())
 {
   delete ptr;
 }
 END_SECTION
 
-ptr = new MRMFeatureSelector();
+ptr = new MRMFeatureSelectorScore();
```

Comment:
Instead of using "global"-like pointers, let's use objects inside test sections. E.g.:
```
START_SECTION(someMethod)
{
  MRMFeatureSelector selector;
  // blabla
}
```

In general, we need to take advantage of all possible positive features of C++11 (and newer) and avoid unsafer features (i.e. pointers). Unless we have good reasons to not do so.

In this case we would also take advantage of block scoping.